src/pages/HomePage/PostsSection.jsx

```diff
@@ -2,16 +2,28 @@ import React, { useEffect } from "react";
 import { MdArrowDropDown } from "react-icons/md";
 import { useDispatch, useSelector } from "react-redux";
 import { PostsWrapper } from "../../components";
-import { fetchUserFeedPosts } from "../../services/posts/postsService";
 
-const PostsSection = () => {
+import {
+  fetchAllPosts,
+  fetchUserFeedPosts,
+} from "../../services/posts/postsService";
+
+
+const PostsSection = ({ type = "all" }) => {
   const posts = useSelector((state) => state.posts);
+  const allPosts = useSelector((state) => state.allPosts);
   const dispatch = useDispatch();
   useEffect(() => {
-    dispatch(fetchUserFeedPosts());
+
+    if (type === "userFeed") {
+      dispatch(fetchUserFeedPosts());
+    } else if (type === "all") {
+      dispatch(fetchAllPosts());
+    }
+
   }, []);
   return (
-    <PostsWrapper posts={posts}>
+    <PostsWrapper posts={type === "all" ? allPosts : posts}>
```

Comment:
Not able to login/signup...please check this whether it is expected behaviour or need to be fixed, code looks clean and understandable...

Reply:
you have to visit `https://zengram.netlify.app/`